pvlib/financial.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+
+def lcoe(production=None, cap_cost=None, fixed_om=None):
+
+    """
+    Levelized cost of electricity as described on pp.43 and 47-48 by [1]
```

Comment:
```suggestion
    Levelized cost of electricity as described on pp.43 and 47-48 by [1]_
```

make this a link to the reference